src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/PlatformManager.java

```diff
@@ -52,12 +69,13 @@ public PlatformManager(List<Platform> crowdPlatforms, WorkerIdentification fallb
 
         //create hashmap of platforms
         platforms = crowdPlatforms.stream()
-                .collect(Collectors.toMap(Platform::getName, Function.identity()));
+                .collect(Collectors.toMap(Platform::getType, Function.identity()));
```

Comment:
but thats a problem! Ok, but i need a non-changing id which can use to identify a platform. when i call `/workers/:platform/identity` i need a string `:platform` which i can use in the worker-service to call the platform. Then the name type is not right.